### PR TITLE
Configurable session timeouts

### PR DESCRIPTION
**Pull request by bcdevlucas**
*Issue #:* /bcgov/entity#5766

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/public/config/configuration.json [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+    "LD_CLIENT_ID": "",
+    "NR_COMPLETION_TIMER_NAME": "nrCompletionTimer",
+    "NR_COMPLETION_TIMEOUT_MS": 0,
+    "DEV_NR_COMPLETION_TIMEOUT_MS": 0,
+    "EXISTING_NR_TIMER_NAME": "existingNrTimer",
+    "EXISTING_NR_TIMEOUT_MS": 0,
+    "DEV_EXISTING_NR_TIMEOUT_MS": 0,
+    "PAYMENT_COMPLETION_TIMER_NAME": "paymentCompletionTimer",
+    "PAYMENT_COMPLETION_TIMEOUT_MS": 120000,
+    "DEV_PAYMENT_COMPLETION_TIMEOUT_MS": 120000
```
Comment (severinbeauvais) [inline client/public/config/configuration.json:19]: Please add check items to the NE Release ticket (/bcgov/entity#5732) to add these to the Dev/Test/Prod config maps for deployment.

**Review by severinbeauvais — APPROVED** [review]
LGTM!

**Review by lekshmimallika-aot — APPROVED** [review]

File: client/public/config/configuration.json (hunk 1 above)
Comment (severinbeauvais) [reply client/public/config/configuration.json:19]: @bcdevlucas Please confirm ^^.

Comment (bcdevlucas) [reply client/public/config/configuration.json:19]: Noted.

Comment (severinbeauvais) [reply client/public/config/configuration.json:19]: @bcdevlucas Please add check items to the NR Release ticket.

(You may not be around when we deploy and we may not remember to do this.)

**Pull request merged**